src/components/LinkItem/LinkItemLits.tsx

```diff
@@ -0,0 +1,14 @@
+import LinkItemWithProfile from './LinkItemWithProfile';
+
+// TODO any 타입 개선
```

Comment:
```suggestion
// FIXME: any 타입 개선
```
아실 수 도 있으시지만 이렇게 적으면 vscode에서 인식해서 하이라이트가 되더라구요!